### DISCUSSION TITLE
Faced 403 Access denied.

### DISCUSSION THREAD

**Discussion by MuruganandamAlwar** [postalserver/postal#3262] [Help with using Postal]
I have installed everything and postal status is up , but i am facing 403 forbidden. Can you please provide any suggestions?

![image](https://github.com/user-attachments/assets/00c99602-f0f9-4a68-8734-56fcb049ad68)
![image](https://github.com/user-attachments/assets/e87aeb4e-fc22-49bb-b8df-5624a97b41dd)
![image](https://github.com/user-attachments/assets/53344585-1531-4b6f-8784-c380c8f05153)

**Comment by willpower232** [comment]
It is a bit particular about the URLs being used, you probably have to use `postal.test.in` or whatever real domain you have in there.

**Reply by MuruganandamAlwar** [reply]
I have tried with domain, faced below issue.

![image](https://github.com/user-attachments/assets/5efae7af-5c44-451a-b1f7-e792c0bca528)

**Reply by willpower232** [reply]
Are you running caddy or nginx?

**Reply by MuruganandamAlwar** [reply]
I am using caddy, below is the config.

![image](https://github.com/user-attachments/assets/0791d4bb-b076-4da3-89c7-7855f0aa0bb1)

**Reply by willpower232** [reply]
I'm not sure what you're trying to set up but you need to use a real subdomain in all places?

![image](https://github.com/user-attachments/assets/e513d717-25ca-4a5e-82cf-b209443d3a93)

**Reply by MuruganandamAlwar** [reply]
I have changed the latest domain in all places in postal dns configuration and in caddy but still faces the issue. Is this issue related to Caddy?

**Reply by willpower232** [reply]
if the DNS is correct then Caddy will generate a certificate for you, if it hasn't then you will have to look at its logs to see what has happened

**Reply by MuruganandamAlwar** [reply]
I am able to lookup my DNS in the localhost on that machine, but still I am facing the same issue. Please provide your suggestions regarding this.

![image](https://github.com/user-attachments/assets/08f56e4b-dae5-4fd4-93c4-36b9b35581c4)
![image](https://github.com/user-attachments/assets/24244ea9-c52f-4735-bfef-f45e8bd7e79c)

**Reply by willpower232** [reply]
you will have to look at caddys logs to see what is going wrong
